### DISCUSSION TITLE
Can I create a Distribution list using exchangelib ??

### DISCUSSION THREAD

**Discussion by parth7676** [ecederstrand/exchangelib#934] [General]
@ecederstrand I am looking to automating the Distribution list creation. Wanted to see I can use exchange lib for that and how ??

**Comment by ecederstrand** [comment]
I don’t think EWS supports creating distribution lists, unfortunately.

**Comment by gismcieri** [comment]
I am not sure if you actually want a distribution list or you want to send to a distribution list but i use exchange lib to send emails to a list of emails.
